source/assets/img/screen-des-styletile.svg

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg version="1.1" viewBox="0 0 270 179" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<title>style tile fenster</title>
+<desc>Created with Sketch.</desc>
```

Comment:
Please compress your SVG, nobody cares about the title or what tool you have used to create the SVG